src/libutil/fs-sink.hh

```diff
@@ -9,27 +9,38 @@
 namespace nix {
 
 /**
- * \todo Fix this API, it sucks.
```

Comment:
This was originally your comment, @edolstra, but I removed it as I am now decently happy with this interface :)